ChangeLog.d/non-psa-pk-implementation.txt

```diff
@@ -0,0 +1,4 @@
+Changes
+   * mbedtls_pk_sign_ext() is now available even without PSA
+     (MBEDTLS_PSA_CRYPTO_C). This improves uniformity in the PK module;
+     it now only depends on MBEDTLS_USE_PSA_CRYPTO for its use of PSA.
```

Comment:
I don't think the end user is really interested in "This improves uniformity in the PK module". This is internal so I would drop that.
Then perhaps it's just me, but the first time I read this sentence it sounded like "mbedtls_pk_sign_ext() is now available even without PSA, [...] now it only depends on MBEDTLS_USE_PSA_CRYPTO" oO. What about rephrasing it as:
"mbedtls_pk_sign_ext() is now always available for RSA as soon as MBEDTLS_RSA_C and MBEDTLS_PKCS1_V21 are defined.". I didn't mention PSA support here because at this time the end user should be familiar with the fact that when USE_PSA is enabled then the PK module uses it.
@mpg wdyt?

Reply:
The "it" referred to the PK module, not `mbedtls_pk_sign_ext()`. But as you managed to interpret it wrong I changed the phrasing (and dropped the comment on the uniformity).